lib/pages/AccountCreation.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:lend_a_hand/services/AccountService.dart';
+import 'package:email_validator/email_validator.dart';
+
+class AccountCreation extends StatefulWidget {
+  @override
+  _AccountCreationState createState() => _AccountCreationState();
+}
+
+class _AccountCreationState extends State<AccountCreation> {
+  final GlobalKey<FormState> _accFormKey = new GlobalKey<FormState>();
+  AccountService account = new AccountService();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Account Sign up'
+        ),
+        centerTitle: true,
+        backgroundColor: Colors.amber,
+      ),
+      body: SafeArea(
+        child: Form(
+          key: _accFormKey,
+          autovalidate: true,
+          child: Column(
+            children: <Widget>[
+              new InputEmail(account: account),
+              new InputName(account: account),
+              new InputPassword(account: account),
+              new SubmitButton(accFormKey: _accFormKey, account: account)
```

Comment:
Seems like `account` is always the same , so could it be setup at the global level? Instead of using a `final AccountService` across all widgets.

Reply:
It seems that the best way to share data between widgets/pages is via the Provider, so initial your login page will call Firebase to return the AuthUser which will then be passed down from page to page. If the account creation page is pushed I am assuming we would acquire necessary data, push to FireBase which would again return the AuthUser which we could then push to the next page?? That piece of code is mostly a placeholder